engine/runtime/src/test/java/org/enso/compiler/EnsoCompilerTest.java

```diff
@@ -1033,14 +1021,6 @@ public void testConstructorMultipleNamedArgs1() throws Exception {
     """);
   }
 
-  @Test
-  @Ignore // Old parser's representation of this is inconsistent with normal treatment of names.
-  public void testConstructorMultipleNamedArgs2() throws Exception {
-    parseTest("""
-    x = (Regex_Matcher.Regex_Matcher_Data case_sensitivity=Case_Sensitivity.Sensitive) dot_matches_newline=True
-    """);
-  }
-
```

Comment:
Intentional difference from the old parser (the old parser would forget it was in type context in some situations, and use a representation of qualified names inconsistent with how it normally handles them in types)